gradio/themes/app.py

```diff
@@ -0,0 +1,139 @@
+import time
+
+import gradio as gr
+
+with gr.Blocks(theme=gr.themes.Default()) as demo:
+    gr.Markdown(
+        """
+    # {THEME} Preview
+
+    To use this theme, set `theme='{AUTHOR}/{SPACE_NAME}'` in `gr.Blocks()` or `gr.Interface`.
```

Comment:
```suggestion
    To use this theme, set `theme='{AUTHOR}/{SPACE_NAME}'` in `gr.Blocks()` or `gr.Interface()`.
````

for consistency?

Reply:
Good catch! Pushing up now